src.ts/key_stores/unencrypted_file_system_keystore.ts

```diff
@@ -43,6 +43,16 @@ async function ensureDir(path: string): Promise<void> {
     }
 }
 
+export async function readKeyFile(path: string): Promise<[string, KeyPair]> {
+    const accountInfo = await loadJsonFile(path);
+    // The private key might be in private_key or secret_key field.
+    let privateKey = accountInfo.private_key;
+    if (privateKey === undefined && accountInfo.secret_key !== undefined) {
```

Comment:
I'd just use `!` (as above instead of `=== null`). You don't really care whether it is empty string, `undefined` or `null`.